tekton/resources/checks/check-pr-has-kind-label.yaml

```diff
@@ -0,0 +1,216 @@
+  - name: pullRequestUrl
+    value: $(body.pull_request.html_url)
+---
+apiVersion: tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: github-pull-request-label-checker
+spec:
+  serviceAccountName: gh-pr-label-checker
+  triggers:
+    - name: trigger
+      interceptors:
+        - cel:
+            filter: "header.match('X-GitHub-Event', 'pull_request') && (body.action == 'opened' || ((body.action == 'labeled' || body.action == 'unlabeled') && body.label.name.contains('kind')))"
+      bindings:
+        - name: trigger-to-check-pr-labels
+      template:
+        name: check-pull-request-labels
+---
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: check-pull-request-labels
+spec:
+  params:
+  - name: labels
+    description: List of labels currently on the Pull Request
+    type: string
+  - name: gitRepository
+    type: string
+  - name: pullRequestUrl
+    type: string
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1alpha1
+    kind: TaskRun
+    metadata:
+      name: check-pr-labels-$(uid)
+    spec:
+      taskSpec:
+        inputs:
+          resources:
+          - name: source
+            type: git
+          - name: pr
+            type: pullRequest
+          params:
+          - name: labels 
+            description: The labels attached to the Pull Request
+          - name: gitRepository
+            description: The URL of the git repository
+          - name: pullRequestUrl
+            description: The URL of the Pull Request
+        outputs:
+          resources:
+          - name: pr
+            type: pullRequest
+        volumes:
+        - name: label-config-v2
+          configMap:
+            name: label-config-v2
+        steps:
+        - name: copy-pr-to-output
+          image: busybox
+          script: |
+            #!/bin/sh
+            mkdir -p $(outputs.resources.pr.path)
+            cp -r $(inputs.resources.pr.path)/* $(outputs.resources.pr.path)/
+        - name: install-pyyaml
+          image: python:3-alpine
+          script: |
+            pip install pyyaml --user
+        - name: check-labels
+          image: python:3-alpine
+          volumeMounts:
+            - name: label-config-v2
+              mountPath: /etc/config
+          script: |
+            #!/usr/bin/env python
+            
+            import json
+            import yaml
+
+            prLabels = json.loads('$(params.labels)')
+            labelNames = list(map(lambda e: e["name"], prLabels))
+            kindLabels = list(filter(lambda e: "kind" in e, labelNames))
+
+            availableLabels = None
+            with open("/etc/config/labels.yaml", "r") as stream:
+              availableLabels = yaml.safe_load(stream)["default"]["labels"]
+
+            availableKindLabels = list(filter(lambda e: "kind/" in e["name"], availableLabels))
+            availableKindNamesAndDescriptions = map(lambda e: "`" +str(e["name"])+ "`" + ":  " + str(e["description"]), availableKindLabels)
```

Comment:
Since we already enforce the list of available labels, I don't believe we need to do this. It should be enough to check that  a label that starts with "kind/" in the list of labels attached to the PR.
@vdemeester thoughts?

Reply:
What if the label starts with `kind/` but isn't in `labels.yaml`?

Reply:
That should not be possible

Reply:
Is this ok to keep then, if `availableKindNamesAndDescriptions` is printed out in the GH comment when the check does not pass?